pkg/api/types.go

```diff
@@ -705,6 +713,8 @@ type ServiceSpec struct {
 	// not be changed by updates.
 	PortalIP string `json:"portalIP,omitempty"`
 
+	Rewrite bool `json:"rewrite,omitempty" description:"Does the load balancer rewrite packets or pass them through"`
```

Comment:
Does this make sense when not used with an external load balancer?

If so, what should the default value be?

If not, it should be put in ServiceStatus in the internal rep (api/types.go) and v1beta3.

Does this PR issue errors for incompatible settings of this field?